docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+name: a12n-server-dev
+
+services:
+  db:
+      image: mysql:8.0
+      restart: always
+      ports:
+        - "3306:3306"
+      environment:
+        - MYSQL_DATABASE=a12nserver
+        - MYSQL_USER=a12nserver
+        - MYSQL_ROOT_PASSWORD=root # NOTE: not recommended for production
+        - MYSQL_PASSWORD='password'
+        - DB_HOST=db
+        - DB_PASSWORD='password'
+        - DB_USER=a12nserver
+        - DB_DATABASE=a12nserver
```

Comment:
Only 1 set is needed, not both! The `MYSQL_` ones are deprecated. Also how would you feel about doing this with postgres instead? I'm personally actually partial to MySQL, but it seems like a lot of people don't take MySQL as serious. They're wrong, but a postgres example might win slightly more people over :'(

Reply:
> The `MYSQL_` ones are deprecated. 
😆  now I realize there is duplication! great spot. I must have been trying to install and run different versions until it clicked and one of them complained about not having `MYSQL_ROOT_PASSWORD`

I have no problem with pgSQL, and actually that was my introduction to SQL.